### ISSUE TITLE
Incompatible with Feather Client capes (And possibly more)?

### ISSUE THREAD

**Issue by Sizerino** [tr7zw/WaveyCapes#28]
![image](https://user-images.githubusercontent.com/94955571/182036705-046c4ffe-9b0b-47cb-8e28-d5dbc6e636dd.png)

Could be incompatible with Badlion, Lunar, and other client given capes...
Can't test with other clients, only have this cape on Feather.

**Comment by tr7zw** [comment]
> Could be incompatible with Badlion, Lunar, and other client given capes...

Can't install mods there, so any mod is "incompatible".

And the Feather developers are aware and working on a fix. Nothing I(or this mod) can do about that.

**Comment by Sizerino** [comment]
Thanks for the reply and sorry about the Badlion and Lunar part I completely forgot they don't have mods lol

**Comment by tr7zw** [comment]
Also noteworthy that Lunar is the reason my mods aren't technically "open source" anymore(due to them using my stuff and in return not treating me fairly), so this mod is more about providing the feature that they want $10+? a month for, for free. So even if you could add it, they probably really wouldn't want you to xD.

**Comment by junkmanner** [comment]
Sorry if this is the wrong place, but can I ask what Lunar did? I assume they violated your mod's licenses, but is there more to it than that?

**Comment by tr7zw** [comment]
Tldr: https://twitter.com/tr7zw/status/1496167049009238017
